community/libxkbcommon/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+mkdir build
```

Comment:
These two lines can be removed.

This is what the `atk` package looks like for reference:

```sh
export DESTDIR="$1"

meson \
    --prefix=/usr \
    --sysconfdir=/etc \
    --mandir=/usr/share/man \
    -Ddocs=false \
    -Dintrospection=false \
    . output

ninja -C output
ninja -C output install
```

Reply:
fixed!